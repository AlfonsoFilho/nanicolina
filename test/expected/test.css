body {
  margin: 0;
  padding: 0;
}
#main {
  margin: 0;
  padding: 0;
}
#main.a {
  margin: 0;
  padding: 0;
}
#main.a .b {
  margin: 0;
  padding: 0;
}
.c {
  display: none;
}
.d {
  margin: 0;
}
div[role='document'] {
  margin: 0;
  padding: 0;
}
.e {
  clear: both;
}
.f,
.g,
.h {
  float: left;
}
.i {
  font-size: 2em;
}
.j {
  text-decoration: underline;
}
.k {
  min-height: 300px;
}
@media all and (max-width: 300px){
  .l {
    display: block;
  }
  .k {
    min-height: 200px;
  }
}
