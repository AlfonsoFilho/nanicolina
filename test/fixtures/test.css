body {
  margin: 0;
  padding: 0;
}
#main {
  margin: 0;
  padding: 0;
}
#main.content {
  margin: 0;
  padding: 0;
}
#main.content .container {
  margin: 0;
  padding: 0;
}
.hidden {
  display: none;
}
.is-home {
  margin: 0;
}
.is-home > .row {
  margin: 0;
}
div[role='document'] {
  margin: 0;
  padding: 0;
}
.row {
  clear: both;
}
.col-xs-12,
.col-sm-8,
.col-lg-4 {
  float: left;
}
.title {
  font-size: 2em;
}
.link {
  text-decoration: underline;
}
.open {
  min-height: 300px;
  visibility: hidden;
}
@media all and (max-width: 300px){
  .visible-xs {
    display: block;
    visibility: visible;
  }
  .open {
    min-height: 200px;
  }
}
