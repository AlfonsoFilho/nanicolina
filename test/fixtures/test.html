<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Test</title>
</head>
<body>
  <div id="main" CLASS="content container" role="document" data-ng-class="{'hidden': isHidden(), 'is-home': isHome()}">
    <div class="row">
      <div class="col-xs-12 col-sm-8 col-lg-4">
        <h1 class="title visible-xs">Title</h1>
        <a id="close" href="http://www.google.com" class="link js-link" ng-class="{'open': isOpen()}">Close</a>
      </div>
    </div>
  </div>
</body>
</html>
