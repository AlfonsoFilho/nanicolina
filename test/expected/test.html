<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Test</title>
</head>
<body>
  <div id="main" CLASS="a b" role="document" data-ng-class="{'c': isHidden(), 'd': isHome()}">
    <div class="e">
      <div class="f g h">
        <h1 class="i l">Title</h1>
        <a id="close" href="http://www.google.com" class="j js-link" ng-class="{'k': isOpen()}">Close</a>
      </div>
    </div>
  </div>
</body>
</html>
